integration-libs/opf/base/root/services/opf-global-message.service.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  StateWithGlobalMessage,
+  Translatable,
+} from '@spartacus/core';
+import { timer } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OpfGlobalMessageService extends GlobalMessageService {
+  protected isGlobalMessageDisabled = false;
+  protected disabledKeys: string[] = [];
+  protected defaultTimeout = 2000;
+  constructor(protected store: Store<StateWithGlobalMessage>) {
+    super(store);
+  }
```

Comment:
[MEDIUM]

No need for explicitly calling constructor - constructor can be dropped entirely

Reply:
[OUTDATED]